src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/judgmentonline/SendJudgmentDetailsCjesHandler.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.judgmentonline;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.camunda.bpm.engine.RuntimeService;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.judgments.CjesService;
+import java.util.List;
+import java.util.Map;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.SEND_JUDGMENT_DETAILS_CJES;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.SEND_JUDGMENT_DETAILS_CJES_SA;
+import static uk.gov.hmcts.reform.civil.enums.YesOrNo.YES;
+import static uk.gov.hmcts.reform.civil.utils.ElementUtils.unwrapElements;
+
+@Service
+@RequiredArgsConstructor
+public class SendJudgmentDetailsCjesHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = List.of(SEND_JUDGMENT_DETAILS_CJES,
+                                                          SEND_JUDGMENT_DETAILS_CJES_SA);
```

Comment:
No changes required for NOTIFY_SET_ASIDE_JUDGMENT Camunda flow? Where is this event SEND_JUDGMENT_DETAILS_CJES_SA called?

Reply:
It this ticket is not necessary changes for notify_set_aside_judgment cause in this ticket we are working only  on Default judgment, the other event is called on another camunda's

Reply:
Understood from Omaira that updating notify_set_aside_judgment flow is not in scope of this ticket and is part of another ticket.